src/connector/src/sink/mod.rs

```diff
@@ -855,9 +859,13 @@ pub trait TwoPhaseCommitCoordinator {
     /// times.
     async fn commit_schema_change(
         &mut self,
-        epoch: u64,
-        schema_change: PbSinkSchemaChange,
-    ) -> Result<()>;
+        _epoch: u64,
+        _schema_change: PbSinkSchemaChange,
+    ) -> Result<()> {
+        Err(SinkError::Coordinator(anyhow!(
+            "Schema change is not implemented for two-phase commit coordinator"
+        )))
```

Comment:
```suggestion
        Err(SinkError::Coordinator(anyhow!(
            "Schema change is not implemented for two-phase commit coordinator {}", std::types::any::<Self>()
        )))
```

can include the type name in the error message.